src/main/kotlin/com/criptext/mail/BaseActivity.kt

```diff
@@ -148,9 +150,19 @@ abstract class BaseActivity: PinCompatActivity(), IHostActivity {
                     )
                 }
                 COMPOSER_MODEL -> {
-                    ComposerModel(
+                    val composerModel = ComposerModel(
                             type = ComposerType.fromJSON(savedInstanceState.getString("composerType")!!, this)
                     )
+                    composerModel.subject = savedInstanceState.getString("subject")!!
```

Comment:
I am not sure about `!!`

Reply:
It's validated at the beginning of the when statement.